about.html

```diff
@@ -0,0 +1,787 @@
+                        </li>
+                      </ul>
+                    </nav>
+                  </div>
+  
+                  <div class="list sub-link">
+                    <ul id="list-47b09bfbb1" class="cmp-list">
+                      <li class="cmp-list__item">
+                        <article>
+                          <a class="cmp-list__item-link" href="/content/leaderimpact/lang/en/leaderhub.html">
+                            <span class="cmp-list__item-title">Leaderhub</span>
+                          </a>  
+                        </article>
+                      </li>
+                      <li class="cmp-list__item">
+                        <article>
+                          <a class="cmp-list__item-link" href="https://give.cru.org/2860269">
+                            <span class="cmp-list__item-title">Give</span>
+                          </a>  
+                        </article>
+                      </li>
+                      <li class="cmp-list__item">
+                        <article>
+                          <a class="cmp-list__item-link" href="/content/leaderimpact/lang/en/contact.html">
+                            <span class="cmp-list__item-title">Contact</span>      
+                          </a>  
+                        </article>
+                      </li>
+                    </ul>
+                  </div>
+  
+                  <div class="container responsivegrid mobile-nav-footer">
+                    <div id="nav-bottom-footer" class="cmp-container">
+                      <div class="container responsivegrid term-policy-wrap">
+                        <div id="container-aa1adff3d8" class="cmp-container">
+                          <div class="container responsivegrid terms-grid">
+                            <div id="container-10bae5eac8" class="cmp-container">
+                              <div class="text term-policy-item">
+                                <div id="text-5b59f11baa" class="cmp-text">
+                                  <p><a href="#">Privacy</a></p>
+                                </div>
+                              </div>
+                              <div class="text term-policy-item">
+                                <div id="text-703f7fc464" class="cmp-text">
+                                  <p><a href="#">Terms of Use</a></p>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+  
+                      <div class="container responsivegrid social-wrap">
+                        <div id="container-8ebb211cec" class="cmp-container">
+                          <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 social-wrap ">
+                            <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
```

Comment:
What is your plan to make these social buttons functional?

Reply:
Can we wrap these social buttons in anchor tag? does that supports AEM ?

Reply:
Yes, as far as I'm aware, we have 3 options here:
1. We use an image with a link
```
<div class="image">
  <div id="image-5c965d7af1" class="cmp-image">
    <a class="cmp-image__link" href="https://facebook.com">
      <img src="./assets/facebook.svg" class="cmp-image__image" itemprop="contentUrl" alt="facebook">
    </a>  
  </div>
</div>
```
3. We use a button with a link and icon
```
<div class="button">
  <a id="button-229994a626" class="cmp-button" href="https://facebook.com">
    <span class="cmp-button__icon cmp-button__icon--icon-facebook" aria-hidden="true"></span>
  </a>
</div>
```
5. We use a text link
```
<div class="text">
  <div id="text-5b9b7eb2bf" class="cmp-text">
    <p>
      <a href="https://facebook.com" target="_blank">
        <img src="/content/dam/LeaderImpact/site-assets/facebook.svg" alt="">
      </a>&nbsp;
    </p>
  </div>
</div>
```

The benefit of the text link is that we can open the link in a new tab.